ledger/sandbox-classic/BUILD.bazel

```diff
@@ -407,9 +407,6 @@ server_conformance_test(
         "--contract-id-seeding=testing-weak",
     ],
     servers = SERVERS,
-    test_tool_args = [
```

Comment:
the test suite is enabled by default. Specifying the includes manually would potentially speed it up since it only runs those tests but I think it’s too fragile and makes it easy to miss newly added test suites that should be included here so I dropped it.